File: src/main/java/net/atos/zac/app/admin/model/RestReferenceTable.java
/*
 * SPDX-FileCopyrightText: 2022 Atos
 * SPDX-License-Identifier: EUPL-1.2+
 */

package net.atos.zac.app.admin.model;

import java.util.List;

public class RestReferenceTable {

    public Long id;

    public String code;

    public String naam;

    public boolean systeem;

    public int aantalWaarden;

    public List<RestReferenceTableValue> waarden;

    public RestReferenceTable() {
    }
}

File: src/main/java/net/atos/zac/app/admin/model/RestReferenceTable.kt
/*
 * SPDX-FileCopyrightText: 2022 Atos, 2024 Lifely
 * SPDX-License-Identifier: EUPL-1.2+
 */

package net.atos.zac.app.admin.model;

import java.util.List;

public class RestReferenceTable {

    public Long id;

    public String code;

    public String name;

    public boolean isSystemReferenceTable;

    public int valuesCount;

    public List<RestReferenceTableValue> values;

    public RestReferenceTable() {
    }
}
